iot_api/user_api/resources/endpoints.py

```diff
@@ -949,8 +949,8 @@ def post(self):
                 )
                 login_attempts.save_to_db()
 
-            access_token = create_access_token(identity=user.username)
-            refresh_token = create_refresh_token(identity=user.username)
+            access_token = create_access_token(identity=user)
+            refresh_token = create_refresh_token(identity=user)
```

Comment:
Why have these lines changed?

Reply:
Because now the entire User object is used to add claims to the user (organization and roles). If we only pass the username, 2 database queries are needed (login + load user info in _add_claims_to_access_token_), against 1 of passing the User object (only 1 database call, in login)